src/tests/end-to-end/test_simple-test.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+import tempfile as tpf
+from distutils.dir_util import copy_tree
+
+import subprocess
+
+import numpy as np
+
+def find_integrity_path(output_dir):
+    op = []
+    for path in Path(output_dir).rglob('checkIntegrity.txt'):
+        op.append(path)
+    assert len(op) == 1
+    return op[0]
+
+def get_interity_check_values(output : Path) -> np.array :
+    integrity_path = find_integrity_path(output)
+    integrity_file = open(str(integrity_path), 'r')
+    output_values = list()
+    for x in integrity_file:
+        output_values.append(float(x))
+    assert len(output_values) == 8
+    return output_values
+
+def launch_solver(study_path, use_ortools = False, ortools_solver = "sirius"):
+    # Find solver executable
+    search_result = list()
+    for path in Path.cwd().parent.parent.parent.rglob('solver/antares-*.*-solver*'):
```

Comment:
I did not understand the  'solver/antares-*.*-solver*'. What does it do exactly ?

Reply:
It looks for any file that matches this regexp. It's slightly more general than the original regex, but still matches exactly 2 files in a fresh build.